corehq/apps/api/resources/auth.py

```diff
@@ -10,25 +10,31 @@
 from corehq.apps.domain.auth import BASIC, determine_authtype_from_header
 from corehq.apps.domain.decorators import (
     api_key_auth,
+    api_key_auth_no_domain,
     basic_auth,
+    basic_auth_no_domain,
     basic_auth_or_try_api_key_auth,
     digest_auth,
+    digest_auth_no_domain,
     login_or_api_key,
     login_or_basic,
     login_or_digest,
-    login_or_oauth2, oauth2_auth)
+    login_or_oauth2,
+    oauth2_auth,
+    oauth2_auth_no_domain,
+)
 from corehq.apps.users.decorators import (
     require_permission,
     require_permission_raw,
 )
 from corehq.toggles import API_THROTTLE_WHITELIST, IS_CONTRACTOR
 
 
-def api_auth(view_func):
+def wrap_4xx_errors_for_apis(view_func):
```

Comment:
in case not obvious, this is a completely different decorator that was also named `api_auth` but is not the one we use everywhere else